persistence/src/commonTest/kotlin/com/wire/kalium/persistence/dao/MemberDAOTest.kt

```diff
@@ -137,6 +139,22 @@ class MemberDAOTest : BaseDatabaseTest() {
         )
     }
 
+    @Test
+    fun givenNonExistingConversation_ThenInsertedOrUpdatedMembersShouldNotBeTriggered() = runTest {
+        val conversationEntity1 = TestStubs.conversationEntity1
+        val member1 = TestStubs.member1
+
+        memberDAO.updateOrInsertOneOnOneMemberWithConnectionStatus(
+            member = member1,
+            status = ConnectionEntity.State.ACCEPTED,
+            conversationID = conversationEntity1.id
+        )
+
+        assertEquals(
+            listOf(), memberDAO.observeConversationMembers(conversationEntity1.id).first()
+        )
```

Comment:
```suggestion
        assertTrue(
           memberDAO.observeConversationMembers(conversationEntity1.id).first().isEmpty()
        )
```

This is just minor stuff, feel free to ignore, just personal preference I think it's more self-explanatory the isEmpty() check 😅